src/datacatalogtordf/agent.py

```diff
@@ -17,30 +17,40 @@
     >>> bool(dataset.to_rdf())
     True
 """
+
 from __future__ import annotations
 
+from dataclasses import dataclass
 from typing import Dict, Optional, Union
 
 from rdflib import Graph, Literal, Namespace, OWL, RDF, URIRef
-from skolemizer import Skolemizer
+from skolemizer import Skolemizer  # type: ignore
 
 from .uri import URI
 
-
 DCT = Namespace("http://purl.org/dc/terms/")
 DCAT = Namespace("http://www.w3.org/ns/dcat#")
 FOAF = Namespace("http://xmlns.com/foaf/0.1/")
 
 
+@dataclass
```

Comment:
Kult med `@dataclass` her. Ser ikkje heilt kvifor vi ikkje kan gjennomføre det for alle?

Reply:
Den har jeg trolig glemt å fjerne igjen.